contracts/scripts/_discoveryprovider-test-init.js

```diff
@@ -54,7 +54,7 @@ const copyBuildDirectory = async (outputPath) => {
   let dir = path.join(__dirname, '..')
   let localTarget = path.join(dir, 'build/contracts')
 
-  await createContractsDir(outputPath)
+  await createDir(outputPath)
```

Comment:
@vicky-g @SidSethi @hareeshnagaraj should we just bite the bullet and rename this file? we've been meaning to for a long time. Maybe `_contractsLocalSetup.js`?

Reply:
yes please 🙏 . `_contractsLocalSetup.js` works

Reply:
👍